src/claimresolver.js

```diff
@@ -102,13 +108,14 @@ class ClaimResolver extends events.EventEmitter {
       var messages = await this.queueService.pollClaimQueue();
       debug("Fetched %s messages", messages.length);
 
-      await Promise.all(messages.map((message) => {
+      await Promise.all(messages.map(async (message) => {
         // Don't let a single task error break the loop, it'll be retried later
         // as we don't remove message unless they are handled
-        return this.handleMessage(message).catch((err) => {
-          debug("[alert-operator] Failed to handle message: %j" +
-                ", with err: %s, as JSON: %j", message, err, err, err.stack);
-        });
+        try {
+          await this.handleMessage(message);
+        } catch (err) {
+          this.monitor.reportError(err, 'warning');
```

Comment:
Before you were including the message in the alert-operator statement, do you not want to include that in the error you're reporting?